README.md

```diff
@@ -991,6 +991,7 @@ Multiplayer game servers and [browser games](https://en.wikipedia.org/wiki/Brows
 - [A Dark Room](https://github.com/doublespeakgames/adarkroom) - Minimalist text adventure game for your browser. ([Demo](https://adarkroom.doublespeakgames.com/)) `MPL-2.0` `HTML5`
 - [EmuLinkerSF](https://emulinker.org) - EmuLinkerSF is an open source Kaillera server. Kaillera is a client/server system that any emulator can implement to enable netplay over the Internet. ([Source Code](https://github.com/God-Weapon/EmuLinkerSF)) `GPL-2.0` `Java`
 - [Hextris](https://github.com/Hextris/hextris) - Fast paced HTML5 puzzle game inspired by Tetris. ([Demo](https://hextris.github.io/hextris)) `GPL-3.0` `HTML5`
+- [EmulatorJS](https://github.com/EmulatorJS/EmulatorJS) - Self-hosted Javascript emulation for various system. `GPL-3.0` `Nodejs`
```

Comment:
- Please add the main site and move the source code to [Source Code] > https://emulatorjs.netlify.app/ 
- Please add the demo > https://emulatorjs.netlify.app/demo.html

Reply:
on it